### PR TITLE
chore(CHANGELOG): add changelog for v1.8.0-RC1

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

**What this PR does / why we need it**:
This PR adds the changelog for v1.8.0-RC1.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,3 +42,27 @@ v1.7.0-RC1 / 2020-02-07
   [#1603](https://github.com/openebs/maya/pull/1603),
   [#1601](https://github.com/openebs/maya/pull/1601),
   [@mittachaitu](https://github.com/mittachaitu))
+
+
+v1.8.0-RC1 / 2020-03-06
+========================
```
Comment (prateekpandey14) [inline CHANGELOG.md]: add above 1.7.0 release

Comment (shubham14bajpai) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
+v1.8.0-RC1 / 2020-03-06
```
Comment (kmova) [inline CHANGELOG.md]: update this with: 
- 1.8.0-RC2  (include changes that went only in 1.8.0-RC2)
- 1.8.0 (all changes from 1.8.0 RC1 and RC2 + additional fix that was only in 1.8.0)

Comment (shubham14bajpai) [reply CHANGELOG.md]: Done

**Pull request merged**